--- openfst-1.5.3/src/extensions/python/Makefile.am.orig	2016-09-02 21:47:46.995731836 +0200
+++ openfst-1.5.3/src/extensions/python/Makefile.am	2016-09-02 21:49:39.489063884 +0200
@@ -2,9 +2,7 @@
 # used to generate them. Consequently, modifications to the .pyx files will not
 # influence the build unless the .cc files are regenerated using Cython.
 
-python_LTLIBRARIES = pywrapfst.la
-
-pyexec_LTILIBRARIES = pywrapfst.la
+pyexec_LTLIBRARIES = pywrapfst.la
 
 pywrapfst_la_SOURCES = pywrapfst.cc
 pywrapfst_la_CPPFLAGS = -I$(srcdir)/../../include $(PYTHON_CPPFLAGS)
